app/src/main/java/com/example/playlistmaker/media/presentation/view_model/NewPlaylistViewModel.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.playlistmaker.media.presentation.view_model
+
+import android.net.Uri
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.playlistmaker.media.domain.db_api.PlaylistsInteractor
+import com.example.playlistmaker.media.domain.local_storage_api.SavePlaylistCoverUseCase
+import com.example.playlistmaker.media.domain.model.Playlist
+import kotlinx.coroutines.launch
+
+class NewPlaylistViewModel(
+    private val playlistsInteractor: PlaylistsInteractor,
+    private val savePlaylistCoverUseCase: SavePlaylistCoverUseCase
+) : ViewModel() {
+
+    private val _isAnythingProvided = MutableLiveData<Boolean>(false)
+    val isAnythingProvided = _isAnythingProvided
```

Comment:
Здесь нужно явно указать тип для проперти, иначе на UI ты получишь MutableLiveData:
```suggestion
    val isAnythingProvided: LiveData<Boolean> = _isAnythingProvided
```

Reply:
А это нужно делать сейчас или вообще? То есть только в этом конкретном случае или всегда?

Reply:
Всегда